mozjs/build/moz.configure/rust.configure

```diff
@@ -92,6 +92,8 @@ def rust_target(rust_compiler, rustc, target, cross_compiling):
             # Android
             ('x86', 'Android'): 'i686-linux-android',
             ('arm', 'Android'): 'arm-linux-androideabi',
+            ('aarch64', 'Android'): 'aarch64-linux-android',
+            ('armv7', 'Android'): 'arm-linux-androideabi',
```

Comment:
No, it's fine to merge them in here. I just want to make sure that they don't get lost as part of the next upgrade we do (which has no definite timeline, unfortunately).